tests/integration-test/src/tests/utils.rs

```diff
@@ -939,3 +956,87 @@ pub fn bpfman_output_map_ids(stdout: &str) -> Vec<String> {
 
     map_ids
 }
+
+#[derive(Debug)]
+pub struct DockerContainer {
```

Comment:
Do we want to be adding a dependency on docker in our core integration tests? Could we just do a 

```
[astoycos@nfvsdn-02-oot bpfman]$ unshare -Urfp --mount-proc

[astoycos@nfvsdn-02-oot bpfman]$ lsns | grep bash
4026533388 pid         1 3416159 astoycos └─-bash

[astoycos@nfvsdn-02-oot bpfman]$ sudo nsenter -t 3416159 -a ls
```

To basically mimic the same thing without docker?

Reply:
To be honest, I don't know.  It would be simpler and probably less overhead to not use it.  I asked in one of the team meetings, and people seemed to be okay with using Docker.  I saw https://www.redhat.com/sysadmin/pid-namespace where they discussed using the command you suggest to "build containers by hand".  Is this exactly equivalent to a Docker container and/or the ones used in k8s, or is it safer to just using Docker?  Personally, I'd have to do some more research to be sure.